### PR TITLE
Quick run_all_manuscript_workflow error catching

### PR DESCRIPTION
**Pull request by cfrick13**
```
====== Successes ======
figure_3_s4_height_density (900.0s) (48.7GB)
figure_4_s8_volume_trajectories (66.4s) (24.1GB)
figure_5_s9_s10_local_growth (256.9s) (38.7GB)
figure_6_s11_compensation (106.4s) (24.1GB)
figure_7_s13_s14_lineage (295.8s) (24.2GB)
figure_s12_feeding_control (149.8s) (21.2GB)
figure_s15_linear_regression_model (939.2s) (24.1GB)
figure_s19_transition_point (18.7s) (24.1GB)
figure_s1_cell_health (18.4s) (21.2GB)
figure_s20_precision_error (14.9s) (5.2GB)
figure_s2_s3_s18_segmentation_model_validation (171.4s) (8.3GB)
figure_s6_s16_inhibitors (242.2s) (15.9GB)
figure_s7_growth_outliers (98.6s) (24.9GB)
====== Failures ======
figure_1_dataset (217.8s) (28.2GB). ValueError setting an array element with a sequence. The requested array has an inhomogeneous shape after 3 dimensions. T                                                  he detected shape was (1, 1, 2) + inhomogeneous part.. See stack trace above.
====== 13 succeeded 1 failed ======
```

### CODE REVIEW

File: nuc_morph_analysis/lib/preprocessing/global_dataset_filtering.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
 def load_dataset_with_features(
     dataset="all_baseline",
     remove_growth_outliers=True,
-    load_local=False,
+    load_local=True,
```
Comment (jcass11) [inline nuc_morph_analysis/lib/preprocessing/global_dataset_filtering.py]:
```suggestion
    load_local=False,
```
Default to not load local would be best!

File: nuc_morph_analysis/lib/preprocessing/global_dataset_filtering.py [hunk 2]
```diff
@@ -393,7 +393,7 @@ def merge_datasets(df_all, df_full):
     'resolution_level',
     '2d_area_cyto',
     'inv_cyto_density',
-    'density'
+    'density',
```
Comment (jcass11) [inline nuc_morph_analysis/lib/preprocessing/global_dataset_filtering.py:396]: Thank you!

File: nuc_morph_analysis/lib/preprocessing/global_dataset_filtering.py [hunk 3]
```diff
@@ -438,6 +438,8 @@ def remove_columns(df, column_list=COLUMNS_TO_DROP):
     df : pandas.DataFrame
         The dataframe with the columns removed.
     """
+    column_list = [col for col in column_list if col in df.columns]
```
Comment (jcass11) [inline nuc_morph_analysis/lib/preprocessing/global_dataset_filtering.py:441]: Thank you!

File: run_all_manuscript_workflows.py [hunk 4]
```diff
@@ -3,6 +3,15 @@
 from nuc_morph_analysis.analyses.colony_area import colony_area_workflow
 from nuc_morph_analysis.analyses.segmentation_model_validation import seg_model_validation_figure_workflow
 
+from nuc_morph_analysis.lib.preprocessing.global_dataset_filtering import load_dataset_with_features
```
Comment (jcass11) [inline run_all_manuscript_workflows.py]: Do we want these changes to be the general behavior? I feel like they were just for a fast test...

**Comment by jcass11** [comment]
I think all the `mypy` errors are independent of the density specific edits I made and are relevant to the broader drafted changes made in the `feature_documentation` branch so I'm okay with leaving them for now and dealing with them in the `feature_documentation` branch after we merge the density deletion changes.

**Comment by cfrick13** [comment]
When I ran it from a fresh .venv 

```
pdm run python run_all_manuscript_workflows.py --only figure_1_dataset

====== Successes ======
figure_1_dataset (419.3s) (46.2GB)
====== 1 succeeded  ======
```

**Pull request merged**